.github/actions/setup/action.yml

```diff
@@ -28,5 +31,10 @@ runs:
     - run: pnpm install
       shell: bash
 
-    - run: pnpm run setup
```

Comment:
`setup` is `pnpm i & pnpm -r run build` and now moved to `if: inputs.skip-tsc == 'false'`